src/esm_runscripts/cli.py

```diff
@@ -280,6 +280,6 @@ def main():
 
     Setup = SimulationSetup(command_line_config)
     # if not Setup.config['general']['submitted']:
-    if not Setup.config["general"]["submitted"] and not no_motd:
-        check_all_esm_packages()
+    #if not Setup.config["general"]["submitted"] and not no_motd:
+    #    check_all_esm_packages()
```

Comment:
Why did you needed to remove this line?

Reply:
To be honest, I don't remember... I am currently trying if it affects my runs